app/javascript/chat/ChatChannelSettings/ActiveMembershipsSection.jsx

```diff
@@ -1,48 +1,53 @@
 import { h } from 'preact';
 import PropTypes from 'prop-types';
+
+import { defaulMembershipPropType } from '../../common-prop-types/membership-prop-type';
 import Membership from './Membership';
 
 const ActiveMembershipSection = ({
   activeMemberships,
   removeMembership,
   currentMembershipRole,
+  toggleScreens,
 }) => {
+  const activeMembershipList = activeMemberships.slice(0, 4);
+
   return (
     <div
       data-testid="active-memberships"
       className="p-4 grid gap-2 crayons-card mb-4"
       data-active-count={activeMemberships ? activeMemberships.length : 0}
     >
       <h3 className="mb-2 active_members">Members</h3>
-      {activeMemberships && activeMemberships.length > 0
-        ? activeMemberships.map((pendingMembership) => (
+      {activeMembershipList && activeMembershipList.length > 0
+        ? activeMembershipList.map((activeMembership) => (
             <Membership
-              membership={pendingMembership}
+              membership={activeMembership}
               removeMembership={removeMembership}
               membershipType="active"
               currentMembershipRole={currentMembershipRole}
               className="active-member"
             />
           ))
         : null}
+      <div className="row align-center">
+        <button
```

Comment:
Use the [design system buttons](https://storybook.dev.to/?path=/story/3-components-buttons--default).

`import { Button } from '@crayons'`